documentation/tutorials/StateAssimilation/TreeRingSDA.Rmd

```diff
@@ -80,7 +80,7 @@ Perform a site-level SIPNET run using the following settings
   </spin.up>
   <forecast.time.step>1</forecast.time.step>
   <start.date>1980/01/01</start.date>
-  <end.date>2014/12/31</end.date>
+  <end.date>2016/12/31</end.date>
```

Comment:
I got this error on pecan2 You asked to pull from the remote 'https://github.com/PalEON-Project/Camp2016.git', but did not specify
a branch. Because this is not the default configured remote
for your current branch, you must specify a branch on the command line.